include/PerformanceBenchmarkingInformation.h

```diff
@@ -0,0 +1,99 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef PerformanceBenchmarkingInformation_h
+#define PerformanceBenchmarkingInformation_h
+
+#include "itkMacro.h"
+#include "itkObject.h"
+#include "itkSimpleFastMutexLock.h"
+
+namespace itk
+{
+/** \class PerformanceBenchmarkingInformation
+ * \brief Static information about this build of the PerformanceBenchmarks
+ *
+ * This class provides a singleton interfacer to a static
+ * std::map< std::string, std::string> of key value pairs
+ * that describe this version of the PerformanceBenchmarks.
+ *
+ * Values in in the map provide information about
```

Comment:
Typo: `Values in in the map provide information about`: `Values in the map provide information about` (duplicate `in`)

Reply:
Fixed.